tests/Extensions/Abstractions.UnitTests/Activities/ActivityExtensionsTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using Microsoft.Omex.Extensions.Abstractions.Activities;
+using Microsoft.Omex.Extensions.Abstractions.Activities.Processing;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Omex.Extensions.TimedScopes.UnitTests
+{
+	[TestClass]
+	public class ActivityExtensionsTests
+	{
+		[TestMethod]
+		public void SetUserHash_SetsHash()
+		{
+			Activity activity1 = new Activity("UserHashTest1");
+			Activity activity2 = new Activity("UserHashTest2");
+
+			string userHash = "some hash value";
+
+			activity1.SetUserHash(userHash);
+
+			Assert.AreEqual(userHash, activity1.GetUserHash());
+			Assert.AreNotEqual(userHash, activity2.GetUserHash());
+		}
+
+
```

Comment:
Nitpick: Inconsistent number of lines between methods

Reply:
Fixed, thank you